presto-common/src/test/java/com/facebook/presto/common/type/BenchmarkTypeSignatureParsing.java

```diff
@@ -0,0 +1,70 @@
205 bigint, field_222 bigint), field_61 row(field_155 varchar))), field_260 row(field_233 boolean, field_119 boolean, field_57 bigint, field_162 bigint, field_14 boolean, field_4 double, field_37 double))), field_109 boolean, field_99 array(row(field_92 bigint, field_54 varchar, field_204 row(field_113 row(field_20 varchar, field_1 bigint, field_92 bigint, field_50 varchar, field_122 array(varchar))), field_12 row(field_86 array(varchar), field_264 varchar), field_225 row(field_100 varchar), field_166 row(field_225 row(field_100 varchar), field_204 row(field_113 row(field_20 varchar, field_137 varchar, field_1 bigint, field_92 bigint, field_175 array(varchar))), field_12 row(field_26 row(field_86 array(row(field_220 varchar)), field_264 row(field_174 varchar))), field_227 row(field_254 row(field_92 bigint, field_206 double, field_123 double, field_235 varchar)), field_216 row(field_182 bigint, field_86 array(row(field_40 varchar)), field_264 varchar), field_81 row(field_176 row(field_30 varchar, field_148 bigint, field_2 varchar)), field_10 row(field_168 varchar), field_247 row(field_19 bigint), field_232 row(field_8 row(field_174 varchar)), field_250 row(field_212 varchar)), field_40 row(field_174 varchar))), field_264 row(field_92 bigint, field_60 varchar), field_104 varchar, field_38 varchar, field_56 varchar, field_196 row(field_92 bigint), field_98 array(row(field_92 bigint, field_155 varchar)), field_120 bigint, field_27 row(field_92 bigint), field_18 row(field_161 boolean), field_130 row(field_92 bigint, field_38 varchar, field_27 row(field_92 bigint)), field_189 row(field_92 bigint, field_116 row(field_1 bigint, field_92 bigint, field_50 varchar, field_137 varchar), field_238 row(field_191 varchar, field_92 bigint, field_235 varchar, field_103 varchar)), field_113 row(field_20 varchar, field_137 varchar, field_1 bigint, field_92 bigint, field_166 row(field_246 row(field_155 varchar), field_7 row(field_234 varchar, field_85 varchar, field_244 varchar, field_72 varchar, field_248 bigint, field_21 varchar, field_172 bigint, field_205 bigint, field_222 bigint), field_146 row(field_74 bigint, field_142 bigint, field_242 bigint, field_155 varchar, field_97 row(field_9 varchar), field_89 bigint), field_61 row(field_74 bigint, field_127 boolean, field_155 varchar)), field_175 array(varchar)), field_26 array(row(field_92 bigint, field_60 varchar)), field_240 row(field_233 boolean, field_119 boolean, field_57 bigint, field_162 bigint, field_14 boolean, field_4 double, field_37 double), field_224 row(field_233 boolean, field_119 boolean, field_57 bigint, field_162 bigint, field_34 boolean, field_207 boolean, field_65 boolean, field_28 bigint), field_43 row(field_215 varchar, field_43 varchar), field_101 row(field_233 boolean, field_119 boolean, field_57 bigint, field_162 bigint, field_14 boolean, field_4 double, field_37 double), field_29 row(field_151 row(field_92 bigint, field_235 varchar), field_135 varchar, field_92 bigint), field_75 boolean))";
```

Comment:
Can we also add test case for signature with enums?

Reply:
added